workflows/edit_eml/set_methods.Rmd

```diff
@@ -12,3 +12,25 @@ eml@dataset@methods <- methods1
 ```
 
 If you want to make minor changes to existing method information that has a lot of nested elements, your best bet may be to edit the EML manually in a text editor (or in RStudio), otherwise there is a risk of accidentally overwriting nested elements with blank object classes, therefore losing method information.
+
+### Adding sampling info to methods section
```

Comment:
This isn't really adding sampling info... it more creating a whole new methods section. I think the ways highlighted above are probably more useful ways to edit methods. This is a valid and sometimes great way to do it as well but this markdown needs a lot more context on when and why to use each method.

Reply:
I haven't explored all the options on how to add sampling info but when I wrote this, I couldn't find any way to just add/update the sampling info when there already was a methods section. And if the methods steps were in a docx, I couldn't find a simple way to read that in for the methods and also include sampling info.